### PR TITLE
Removed env from redirect and changed redirect from method to macro.

### PR DESCRIPTION
**Pull request by ilatif**
Removed `env.redirect` in-favor of `redirect` in-order to make Kemal's
API look more like Sinatra.

Thanks @sdogruyol and @f for introducing me to macros :-).

### CODE REVIEW

File: src/kemal.cr [hunk 1]
```diff
@@ -17,13 +20,12 @@ at_exit do
   config.add_handler Kemal::Handler::INSTANCE
   config.add_handler HTTP::StaticFileHandler.new("./public")
 
-  server = HTTP::Server.new("0.0.0.0", config.port, config.handlers)
```
Comment (sdogruyol) [inline src/kemal.cr]: We need to bind to `0.0.0.0`

Comment (ilatif) [inline src/kemal.cr]: Oh... But I pushed my code after pulling updates from your branch and I do remember that you bind it to `0.0.0.0` in one of your previous commits. What happened to that particular line?

Comment (ilatif) [inline src/kemal.cr]: Oh... I just checked `kemal.cr` on `master` branch and that `0.0.0.0` line is not there as well.

Comment (sdogruyol) [inline src/kemal.cr]: Woah there must have been some merge override.

Comment (ilatif) [inline src/kemal.cr]: Yup... But I didn't do that because my this code hasn't been merged yet :speak_no_evil:.

Comment (sdogruyol) [inline src/kemal.cr]: Yeah i know :+1: It must have been a mistake by me

Comment (ilatif) [inline src/kemal.cr]: :smile:, no worries. Pull Request looking fine else-wise?

Comment (sdogruyol) [inline src/kemal.cr]: Yeah could you please bind it to

Comment (ilatif) [inline src/kemal.cr]: Done :+1:.

**Comment by sdogruyol** [comment]
Thanks. LGTM :+1:

**Comment by ilatif** [comment]
Thanks :clap:.

**Pull request merged**